src/main/java/cn/nukkit/block/BlockTrappedChest.java

```diff
@@ -98,7 +98,7 @@ public int getWeakPower(BlockFace face) {
             playerCount = ((BlockEntityChest) blockEntity).getInventory().getViewers().size();
         }
 
-        return playerCount < 0 ? 0 : playerCount > 15 ? 15 : playerCount;
+        return playerCount < 0 ? 0 : Math.min(playerCount, 15);
```

Comment:
Clamp should be used here instead

Reply:
My bad. We can remove the first part here since it will never be 0.